### PR TITLE
count hits distinct per useragent_id

### PR DESCRIPTION
**Pull request by SimonLab**
ref: #61

### CODE REVIEW

**Comment by nelsonic** [comment]
@SimonLab I think it's worth adding a small paragraph to the `README.md` informing people that they will only see the count increase if they visit from a new/different web browser this is to avoid counting the same person as multiple visits. 💭

**Comment by SimonLab** [comment]
@nelsonic I think the PR is ready for review.
The count will now only returns unique user (ie ip and user agent different).

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -0,0 +1,46 @@
+
```
Comment (nelsonic) [inline .github/workflows/ci.yml:1]: Thanks for adding GitHub workflow. 👍 
We'll need to update the badge in the `README.md` at some point. 💭

**Review by nelsonic — APPROVED** [review]
Merci @SimonLab 👍

**Comment by SimonLab** [comment]
@nelsonic Just to check, I don't think the latest version of hits has been deployed and doesn't contained this new code, am I right?

**Comment by nelsonic** [comment]
@SimonLab yeah, we don't yet have Continuous Deployment for Hits https://github.com/dwyl/learn-devops/issues/77 
apologies for not making that clear ... it means your work has not yet been deployed. 

> The slightly longer story is that the computer I deployed it to Fly.io from a few weeks ago has decided to stop working ... 
so I have to re-authenticate which will take a few more steps ...

I will (a) deploy it manually tonight (**`T10m`**) and (b) work on CI/CD (est: **`T2h`**) tomorrow morning.
I'm keen to build more _useful_ features for this project https://github.com/dwyl/product-roadmap/issues/7 
e.g. ***Graphs*** with Phoenix LiveView + `{ChartLibraryTBD}`

**Comment by nelsonic** [comment]
The code in this PR has been deployed to Fly.io via: https://github.com/dwyl/hits/pull/133
https://github.com/dwyl/phoenix-liveview-chat-example
hits-one-per-browser

However when I visit the page in a totally different browser e.g. Firefox, I still only see the **hits: 3**:
hits-liveview-firefox

Can't say I understand why ... 🤷‍♂️ 
@SimonLab any ideas?

**Comment by SimonLab** [comment]
It's stuck at 3 also for me at the moment.
Is there a way to access the database to see what values are saved?

**Comment by nelsonic** [comment]
Good question. I think there is. Let me dig in and try and share. ⏳

**Pull request merged**